src/apps/lwaftr/lwaftr.lua

```diff
@@ -19,9 +19,20 @@ local ffi = require("ffi")
 local band, bnot, rshift = bit.band, bit.bnot, bit.rshift
 local bitfield = lib.bitfield
 local C = ffi.C
+local cast = ffi.cast
+local receive, transmit = link.receive, link.transmit
+local empty, full = link.empty, link.full
```

Comment:
`link.full` is not finally used, it was previously used in the while loops but commented.